### PR TITLE
Remove NumericRange query type

### PR DESCRIPTION
**Pull request by metamaker**
NumericRange query is deprecated in ElasticSearch since version 0.90.8.
Use Query\Range instead of Query\NumericRange.

@see https://www.elastic.co/guide/en/elasticsearch/reference/0.90/query-dsl-numeric-range-filter.html

### CODE REVIEW

**Comment by ruflin** [comment]
Could you please update the CHANGELOG.md file?

**Comment by metamaker** [comment]
Yes, of course. I somehow missed that statement in CONTRIBUTING.md, sorry :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ All notable changes to this project will be documented in this file based on the
 ### Improvements
 
 ### Deprecated
-
+ - Removed `Query\NumericRange`, use `Query\Range` instead [#1334](https://github.com/ruflin/Elastica/pull/1334)
```
Comment (ruflin) [inline CHANGELOG.md]: This probably goes more under BC Breaks as we didn't deprecated it but removed it.

Comment (metamaker) [inline CHANGELOG.md]: Ok, moved.

**Comment by ruflin** [comment]
@metamaker Merged. Thanks for the contribution.

**Pull request merged**